config.txt

```diff
@@ -77,6 +77,16 @@ null,
 "nicehash_nonce" : false,
 
 /*
+ * Manual hardware AES override
+ *
+ * Some VMs don't report AES capability correctly. You can set this value to true to make the miner use hardware
+ * AES or to false to make it use software AES.
+ * 
+ * WARNING: setting this to true on a CPU that doesn't support hardware AES will crash the miner.
+ */
+"aes_override" : null,

```

Comment:
`You can set this value to true to make the miner use hardware AES or to false to make it use software AES.` Any idea if that can be further clarified?